automerge-backend/src/backend.rs

```diff
@@ -146,7 +153,12 @@ impl Backend {
     }
 
     pub fn merge(&mut self, remote: &Backend) -> Result<Patch, AutomergeError> {
-        let missing_changes = remote.get_missing_changes(self.op_set.clock.clone());
+        let missing_changes = remote
+            .get_missing_changes(self.op_set.clock.clone())
+            .iter()
+            .cloned()
+            .cloned()
```

Comment:
Do we need to clone twice?

Reply:
Oh thats right - i had forgotten about this... Fiddled with some lifetimes and managed it with no cloning at all!